projects/angular/src/utils/variables/API.md

```diff
@@ -0,0 +1,35 @@
+# Global properties-clarity - Style API
+
+| CSS Variable Name          | Description                                                        |
+| -------------------------- | ------------------------------------------------------------------ |
+|  --clr-global-borderradius | It changes the radius of the elemnt's corner and is applied globally.
+|  --clr-global-borderwidth | It changes the width of the border and is used globally.
+|  --clr-global-app-background | It changes the background colour of the body and is used globally.
```

Comment:
colour -> color 
for consistency
same with other items below

Reply:
Updated, As suggested